src/hnsw/utils.h

```diff
@@ -6,11 +6,15 @@
 #include "options.h"
 #include "usearch.h"
 
+extern bool versions_match;
```

Comment:
do these need to be exported? Isn't the `VersionMatch()` the only API used externally?